### PR TITLE
feat(bivariate-matrix): 11607 - show ui response tooltip 

### PR DESCRIPTION
**Pull request by propakov**
- fade animation is added to features/tooltip constantly

https://kontur.fibery.io/Tasks/Task/UI-response-for-selecting-layers-11607

Screenshot 2022-09-05 at 17 21 53

### CODE REVIEW

File: src/features/bivariate_manager/components/ConnectedBivariateMatrix/ConnectedBivariateMatrix.tsx [hunk 1]
```diff
@@ -37,151 +39,161 @@ const mapHeaderCell = (
...
+      resetTooltip();
+      setTooltip({
+        popup: <ProgressTooltip close={resetTooltip} />,
+        position,
+        initiatorId: TOOLTIP_ID,
+        hoverBehavior: true,
+      });
+    }, 400),
+    [resetTooltip, setTooltip],
+  );
```
Comment (IlyaIzr) [inline src/features/bivariate_manager/components/ConnectedBivariateMatrix/ConnectedBivariateMatrix.tsx:123]: Can you explain why do you use debounce?

File: src/features/bivariate_manager/components/ProgressTooltip/ProgressTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M8.98598 2.74142C9.05221 2.38823 8.81918 2.04465 8.46088 2.01744C8.30877 2.00588 8.15507 2 8 2C4.68629 2 2 4.68629 2 8C2 11.3137 4.68629 14 8 14C11.3137 14 14 11.3137 14 8C14 7.72291 13.9812 7.45021 13.9449 7.1831C13.8964 6.82712 13.5395 6.61511 13.191 6.70225C12.8428 6.78929 12.6358 7.14218 12.6736 7.49906C12.6911 7.66365 12.7 7.83078 12.7 8C12.7 10.5957 10.5957 12.7 8 12.7C5.40426 12.7 3.3 10.5957 3.3 8C3.3 5.40426 5.40426 3.3 8 3.3C8.07433 3.3 8.14825 3.30173 8.22174 3.30514C8.58011 3.32178 8.91987 3.09403 8.98598 2.74142Z"
+      fill="currentColor"
+    />
+  </svg>
+);
+
```
Comment (IlyaIzr) [inline src/features/bivariate_manager/components/ProgressTooltip/ProgressTooltip.tsx]: I believe you should ask designer to add icon into icons pack

File: src/features/tooltip/index.tsx [hunk 3]
```diff
@@ -1,8 +1,22 @@
 import { useAtom } from '@reatom/react';
 import { currentTooltipAtom } from '~core/shared_state/currentTooltip';
+import {
+  CSSTransitionWrapper,
+  fadeClassNames,
+} from '~features/bivariate_color_manager/components/CssTransitionWrapper/CssTransitionWrapper';
```
Comment (IlyaIzr) [inline src/features/tooltip/index.tsx]: ❗ import into feature from another feature

Comment (propakov) [reply src/features/tooltip/index.tsx]: oh, really  :/ 
nice catch! 👁️ 💎

File: src/features/bivariate_manager/components/ConnectedBivariateMatrix/ConnectedBivariateMatrix.tsx (hunk 1 above)
Comment (propakov) [reply src/features/bivariate_manager/components/ConnectedBivariateMatrix/ConnectedBivariateMatrix.tsx:123]: same reason of using in inputs, here we can select cells/heading fast, we don't need this tooltip on every select, just in the end.

**Review by IlyaIzr — APPROVED** [review]

**Review by Akiyamka — APPROVED** [review]

**Pull request merged**